### PR TITLE
Don't run non-tvm_op GraphRuntime nodes in Debug Runtime over RPC.

### PR DESCRIPTION
**Pull request by areusch**
* These are filtered out in SetupOpExecs for normal debug runtime operation.
 * Fixes use of debug runtime with µTVM.

@tmoreau89 @tqchen @jroesch @tkonolige

### CODE REVIEW

**Comment by tkonolige** [comment]
Is this for ops that are extern calls? Like calling into MKL?

**Comment by areusch** [comment]
@tkonolige no, the GraphRuntime has two different paths to run operators: one for local operators where non-kernel ops could be done fairly inexpensively (I.e. layout transforms maybe? this isn't actually used) and one for RPC operators (where that is not easy). right now tvm_op == "null" (for graph nodes which are inputs) or "tvm_op" (for graph nodes which are operator function execs).

File: src/runtime/graph/debug/graph_runtime_debug.cc [hunk 1]
```diff
@@ -110,6 +110,16 @@ class GraphRuntimeDebug : public GraphRuntime {
   }
 
   double RunOpRPC(int index, int number, int repeat, int min_repeat_ms) {
+    if (nodes_[index].op_type != "tvm_op") {
```
Comment (tkonolige) [inline src/runtime/graph/debug/graph_runtime_debug.cc:113]: Could you just leave a comment for why we are doing this

**Review by tkonolige — APPROVED** [review]

File: src/runtime/graph/debug/graph_runtime_debug.cc (hunk 1 above)
Comment (areusch) [reply src/runtime/graph/debug/graph_runtime_debug.cc:113]: done

**Comment by areusch** [comment]
@tqchen

**Review by tmoreau89 — APPROVED** [review]
LGTM

**Comment by tmoreau89** [comment]
Thanks @tkonolige @areusch the PR has been merged

**Pull request merged**